dts/bindings/sensor/st,ism330dhcx-common.yaml

```diff
@@ -22,6 +42,26 @@ properties:
         -  4 #  4g (0.122 mg/LSB)
         -  2 #  2g (0.061 mg/LSB)
 
+    gyro-odr:
+      type: int
+      required: false
+      default: 0
+      description:
+          Specify the default gyro output data rate expressed in samples per second (Hz).
+          Default is power-up configuration.
+      enum:
+        - 0  # Power-Down
```

Comment:
Might want to consider the same idea here.

Reply:
ack